docs/open_source/scan/scan_vision/index.md

```diff
@@ -0,0 +1,198 @@
+# 📸 Vision model scan
+
```

Comment:
in this page, could you make the wrapping of dataset and model more generic? like we did here: https://docs.giskard.ai/en/stable/open_source/scan/scan_tabular/index.html just give an example on how one would wrap their dataset and what they minimally need to write in terms of methods.

for dataloader for instance it's:
```py
    @abstractmethod
    def get_image(self, idx: int) -> np.ndarray:
```

for models, it's:
```py
   @abstractmethod
    def predict_image(self, image: np.ndarray) -> np.ndarray:
        """abstract method that takes one image as input and predicts its landmarks as an array

        Args:
            image (np.ndarray): input image
        """

        ...
```

Could you also give an example on how to write a Dataloader wrapper to do a sort of filtering for instance?